urlhaus/info.json

```diff
@@ -0,0 +1,385 @@
+{
+    "name": "urlhaus",
+    "version": "1.0.1",
+    "label": "Url Haus",
+    "description": "URLhaus is a project operated by abuse.ch. The purpose of the project is to collect, track and share malware URLs, helping network administrators and security analysts to protect their network and customers from cyber threats.",
+    "publisher": "Community",
+    "icon_small_name": "urlhaus.png",
+    "icon_large_name": "urlhaus.png",
+    "category": "Threat Intelligence",
+    "configuration": {
+        "fields": [
+            {
+                "title": "Server URL",
+                "type": "text",
+                "name": "server_url",
+                "required": true,
+                "visible": true,
+                "editable": true,
+                "value": "https://urlhaus-api.abuse.ch"
+            }
+        ]
+    },
+    "operations": [
+        {
+            "operation": "get_hash_details",
+            "title": "Get Hash Details ",
+            "description": "To retrieve information about an Hash",
+            "category": "investigation",
+            "annotation": "get_hash_details",
+            "output_schema": {
+                "query_status": "",
+                "md5_hash": "",
+                "sha256_hash": "",
+                "file_type": "",
+                "file_size": "",
+                "signature": "",
+                "firstseen": "",
+                "lastseen": "",
+                "url_count": "",
+                "urlhaus_download": "",
+                "virustotal": "",
+                "imphash": "",
+                "ssdeep": "",
+                "tlsh": "",
+                "urls": [
+                    {
+                        "url_id": "",
+                        "url": "",
+                        "url_status": "",
+                        "urlhaus_reference": "",
+                        "filename": "",
+                        "firstseen": "",
+                        "lastseen": ""
+                    }
+                ]
+            },
+            "parameters": [
+                {
+                    "title": "Hash name",
```

Comment:
The first letter of every word in the title needs to be capital so, please change it to Hash Name.
And do the same for every title.

Reply:
Change has been done for all items